test/appIcon.js

```diff
@@ -22,7 +23,8 @@ describe('appIcon', function () {
       monochrome: false,
       inverted: false,
       darkMode: true,
-      platform: 'linux'
+      platform: 'linux',
+      remainingModeString: ''
```

Comment:
There should be test that will make sure under different remaining minutes we get the correct icons for each platform, similar as it is done with dark mode etc. 

There should also be test that when break is on, the icon is not "1", or "0" but the regular one. In fact, we should never see icon "0".

Reply:
Done. Cases with pause I not check coz they in first if. But if their really needed they could be done.
Also move all logic to AppIcon can be done easly.

Reply:
Do you wanna try to move it? I think it would be better

Reply:
Done.